pipelines/combine-zarr-sample-genotypes/Snakefile

```diff
@@ -17,32 +19,34 @@ if "contigs" not in config:
 rule all_merge:
   input:
     expand(
-      os.path.join("{sampleset}/callset.zarr/{chrom}/calldata/{field}"),
+      os.path.join("{samplesets_output_dir}/{sampleset}/callset.zarr/{chrom}/calldata/{field}"),
```

Comment:
Looks great Lee- thanks. 

I only have one comment- which is that the `sample_sets_output` dir variable can be confusing. 

I would remove line `5` and at lines `23`, `27`, and `35` can we use `config["sample_sets_output]` instead of the variable.

All other mentions of `samplesets_output_dir` are in wildcards eg:

```
zarr=directory("{samplesets_output_dir}/{sampleset}/callset.zarr/{chrom}/calldata/{field}"), 
```
and can be left as is.

Reply:
I see. You're saying it's unnecessary to store `config["samplesets_output_dir"]` in `samplesets_output_dir`, where we can just use `config["samplesets_output_dir"]` instead, to make it clearer that it's coming from config.
OK, will do.

Reply:
It does seem to become subjective as to whether this change actually makes the code clearer though, because `x["y"]` is harder to read than `x`. Also `config["samplesets_output_dir"]` can't _simply_ replace `samplesets_output_dir` in string templates like `manifest="{samplesets_output_dir}/{sampleset}/manifest"`.
I can probably work around the double quotes, but it wouldn't look as simple.

Reply:
Having said that, it is consistent with the rest of the Snakemake file, and would reduce the number of global variables (outside the config object), so I'm like totally down with it.

Reply:
There might be some existing baseline confusion over wildcards vs variable names, which would be good to extinguish, e.g. with `sampleset`:
```
sampleset = os.path.basename(os.path.dirname(fofn_path))
[...]
    expand(
      os.path.join("foobar/{sampleset}/callset.zarr/{chrom}/calldata/{field}"),
      sampleset=[sampleset],
```
1. Does `sampleset=[sampleset],` need the square brackets? Is it even needed at all?
2. What is the story with scope here? Is the `{sampleset}` value coming from the wildcard or the global variable?

Reply:
The `qc/Snakefile` just uses the existing `sampleset` global variable when expanding the path, so I reckon the `sampleset` "wildcard" is superfluous here.
Will remove and test.

Reply:
Got error, so I'll roll back and reconsider
```
WildcardError in line 17 of /home/lee/gitRepos/vector-ops/vector-tools/pipelines/combine-zarr-sample-genotypes/Snakefile:
Wildcards in input files cannot be determined from output files:
'sampleset'
```

Reply:
Fixed by removing `sampleset` as wildcard, using the var instead.